youtube_dl/extractor/youtube.py

```diff
@@ -308,6 +308,77 @@ def _extract_ytcfg(self, video_id, webpage):
                 r'ytcfg\.set\s*\(\s*({.+?})\s*\)\s*;', webpage, 'ytcfg',
                 default='{}'), video_id, fatal=False)
 
+    def _search_results(self, query, params):
```

Comment:
The code for checking the number of entries is dropped in favor of doing it with itertools.slice below. Otherwise, this function is untouched - only renamed and moved

Reply:
Yes, much more useful there.